controllers/provider-gcp/pkg/apis/gcp/helper/helper.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helper
```

Comment:
Perhaps it's better to put this in the `internal` package? It's there already and meant to contain similar helper functions that are shared among controllers (I assume this code will be shared with other controllers, otherwise I think it should have stayed unexported in the controller package).

Reply:
Hm, to me this looks rather like something that is highly API specific, that's why I put it to the API helpers. However, it's the internal API, so nobody other than the provider-aws will (should) use this anyways, so from that PoV the `internal` package is a better fit. Will move.